lib/puppetserver/ca/action/list.rb

```diff
@@ -57,6 +60,12 @@ def run(input)
           config = input['config']
           certnames = input['certname'] || []
           all = input['all']
+          output_format = input['format']
+          
+          if output_format != nil && output_format != 'json' 
```

Comment:
`output_format` here would be valid as either 'json' or 'text', so the validation probably should be more like ['text', 'json'].include?(output_format)` rather than the `!= 'json'`.

Reply:
might even be useful to have a `valid_output_format?` method that does this check